src/controller/nonlinmpc.jl

```diff
@@ -204,7 +205,7 @@ function NonLinMPC(
     L_Hp = diagm(repeat(Lwt, Hp)),
     Cwt  = DEFAULT_CWT,
     Ewt  = DEFAULT_EWT,
-    JE::Function = (_,_,_) -> 0.0,
+    JE::Function = (_,_,_,_) -> 0.0,
```

Comment:
In situations like this, I like `(args...)->0.0` to be future proof :)

Reply:
Excellent idea!!!